src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsRefreshing.java

```diff
@@ -14,11 +14,12 @@
           "108480239532"};
   private static final String WIKI_NAME = "project43";
   private static final String PATH = "SyntheticTests/Slots/RefreshOnView";
+  private static final String ARTICLE_MIDDLE_SECTION_SELECTOR = "#ArticleMidSection.mw-headline";
 
   private void scrollToSeeFMR(String slotName, AdsBaseObject page) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "scrollToSeeFMR" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)